RosNet/DataModel/RosBag.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using System.Collections.Generic;
 
+using RosNet.Field;
+
```

Comment:
Please extract the public members of this class to an Interface, same for your datamodels